test/test_data/cairo.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2022 Orange
```

Comment:
You might want the copyright here to be Keysight as in the one above.  I'll mark this as requiring changes so I don't merge it by mistake before you're ready.

Reply:
Good catch, thanks! I updated the copyright statement and rebased on main, should be good now.